src/System.IdentityModel.Tokens.Jwt/JwtPayload.cs

```diff
@@ -488,7 +488,7 @@ public void AddClaims(IEnumerable<Claim> claims)
             }
         }
 
-        internal static object GetClaimValueUsingValueType(Claim claim)
+        private static object GetClaimValueUsingValueType(Claim claim)
```

Comment:
non of these internals seems to be referenced anywhere, so they are converted to private